### DISCUSSION TITLE
Who's got the M5Stack Atom S3 working?

### DISCUSSION THREAD

**Discussion by tmchow** [echavet/MitsubishiCN105ESPHome#143] [Q&A] [answered]
I've been trying out the M5stack Atom S3, after failing with the S3 **Lite**.  Wondering if anyone has the Atom S3 (non-Lite) working with `esp-idf` and can share their config?

I've found I can install it with `type: arduino` but not having luck on the UART pins for Rx and Tx. Just don't know the mapping.

I found this but it's non specific to get the Rx and Tx working: https://github.com/esphome/feature-requests/issues/2332#issuecomment-1712933090

Hoping someone here has a working config.

**Comment by klim123123** [comment, marked as answer]
[please check this link](https://github.com/echavet/MitsubishiCN105ESPHome/discussions/83)
